features/admin.users.v1/models/user.ts

```diff
@@ -345,17 +345,12 @@ export interface PayloadInterface {
       schemas: string[];
 }
 
-/**
- * Type of the bulk user import operation status.
- */
-export type BulkUserImportOperationStatus = "Success" | "Failed" | "Warning";
-
 /**
  * Interface for the bulk user import operation response.
  */
 export interface BulkUserImportOperationResponse {
     resourceIdentifier: string;
-    status: BulkUserImportOperationStatus;
+    status: string;
```

Comment:
import BulkUserImportStatus from "../constants/user-management-constants"

```suggestion
    status: BulkUserImportStatus;
```

Reply:
Here, the status is provided as i18n translation values [1], so specifying a strict type is not necessary. Additionally, the status is only used for display purposes and not for any logic processing. Therefore, the type BulkUserImportOperationStatus has been replaced with a generic string.

[1] - https://github.com/PasinduYeshan/wso2-identity-apps/blob/964720e75d6c2b9dd7a71a48361e446f2448c344/features/admin.users.v1/components/wizard/bulk-import-user-wizard.tsx#L1414